Add Dashboard Component along with Canvas Charts

CajetanRodrigues opened the pull request:
Fix #12 

### Balsamiq design

https://openmf.mybalsamiq.com/projects/onlinebankingapp30/DashboardComponent

![dashboard_balsa](https://user-images.githubusercontent.com/37682760/62362169-0cdc3700-b53a-11e9-8a6e-354c731769ec.PNG)

### UI

- Added mobile responsive blocks
- Added Canvas charts for the visualization of data

Upper Part

![dashboard](https://user-images.githubusercontent.com/37682760/62362445-90962380-b53a-11e9-97e2-e4502b5b5cf6.PNG)

Lower Part

![dashboard2](https://user-images.githubusercontent.com/37682760/62362449-92f87d80-b53a-11e9-8150-852e0139ffe4.PNG)

ankit01ojha commented:
@CajetanRodrigues squash your commits here too.

ankit01ojha reviewed (commented):
@CajetanRodrigues , could you just squash these commits into one.

CajetanRodrigues committed:
add dashboard component


add Canvas charts to dashboard


update dashboard


update dashboard component


update dashboard component


add loan service


add savings/shares service


update dashboard component


update dashboard component


add canvas.js locally

CajetanRodrigues commented:
> @CajetanRodrigues , could you just squash these commits into one.

Done

CajetanRodrigues committed:
Merge branch 'master' into dashboard
